src/services/shiftRotaService.ts

```diff
@@ -17,6 +17,26 @@ export default class ShiftRotaService {
         return this.shiftRepository.getShiftForToday();
     }
 
+    getNextWeekShifts =  async () => {
+
+        
+        const days =[1,7,6,5,4,3,2];
+        const day1 = new Date();
+        let date1 = null;
+        let date2 = null;
+        if(day1.getDate() == 1) {
+           date1 = day1;
+        }
+        else {
+           date1 = day1.setDate(day1.getDate()+days[day1.getDay()]);
+           date2 = day1.setDate(day1.getDate()+days[day1.getDay()]-3);
+        }
+        
+        let date3 = new Date(date1).toISOString().substring(2).split('T')[0];;
+        let date4 = new Date(date2).toISOString().substring(2).split('T')[0];;
+        return this.shiftRepository.getShiftsForDateScope(date3,date4);
+    }
```

Comment:
I don't check the logic here as there's a lot of stuff happening :D I trust all is fine (but it should be covered with tests - then I could check the test, see what's the expected value, run it, add some console.logs there if needed etc.)

Reply:
ok, i remember about tests - so not resolving this conversation for now